src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -313,6 +317,10 @@ private static TwoWindingsTransformerMapData toMapData(TwoWindingsTransformer tr
         if (limits2 != null && !Double.isNaN(limits2.getPermanentLimit())) {
             builder.permanentLimit2(limits2.getPermanentLimit());
         }
+        BranchStatus branchStatus = transformer.getExtension(BranchStatus.class);
```

Comment:
Sonar and IntelliJ signal a "Raw use of parameterized class 'BranchStatus' ".
You could be more precise with BranchStatus<TwoWindingsTransformer> that makes IntelliJ happy.
Though rest of the code in this source is riddled with such unprecisions